theme/static/js/hs-vue/notifications-app.js

```diff
@@ -82,6 +82,15 @@ $(document).ready(function () {
                         "failed": "Failed",
                         "delivered": "Delivered"
                     }
+                },
+                "resource delete": {
+                    title: "Resource delete",
+                    status: {
+                        "progress": "In progress...",
+                        "completed": "Completed",
+                        "failed": "Failed",
+                        "delivered": "Delivered"
+                    }
                 }
```

Comment:
while you're in the notifications-app can you fix a cross browser issue and ensure all methods follow the `exampleFunc: function() {` pattern, including spacing? Some versions of Microsoft and potentially others will crash when the function() is missing. I learned this as a part of work @hydrocheck is doing with QA. There's at least one, possibly more.

Reply:
I don't quite get what needs to be changed here for the cross browser issue. If you give me guidance with an example, I can fix them for notifications-app while I am at it in this PR.

Reply:
scheduleCheck and checkStatus are missing `:function() {` and breaking the page for some Windows/IE-Edge combinations

Reply:
![image](https://user-images.githubusercontent.com/45397838/108880562-6995bb80-75d0-11eb-8872-76ad348411b5.png)

Reply:
I'll be glad when we move further away from Django and can use automatic linting and fixing. I believe this would be very usefule @hydrocheck 
![image](https://user-images.githubusercontent.com/45397838/108880729-9053f200-75d0-11eb-81d7-c046cd2af45e.png)

Reply:
@cuahsimarko Have just made a commit to add the missing function() in a couple of places. Cannot find any more such missing function() cases in this notification app js file, so hopefully this is good enough to address cross browser issue you noted above.

Reply:
Yes, it's that easy. If it were more complicated, I would have just fixed it and opened a PR based off of your PR for you.